core/src/main/java/org/radargun/reporting/AbstractReporter.java

```diff
@@ -0,0 +1,30 @@
+package org.radargun.reporting;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.radargun.config.MasterConfig;
+import org.radargun.config.Property;
+import org.radargun.logging.Log;
+import org.radargun.logging.LogFactory;
+
+public abstract class AbstractReporter implements Reporter {
+
+   private static Log log = LogFactory.getLog(AbstractReporter.class);
+
+   @Property(doc = "Which master return code should make the report run. Default is all. Comma separated.")
```

Comment:
@diegolovison from our discussion, I thought this property would be a `boolean` named `skipOnStageFailures`?

Reply:
@alanfx yes. it is a boolean. https://github.com/radargun/radargun/pull/617/files#diff-d2f544a284d57c18ed2c8ecb87a549ebR20

Reply:
@diegolovison OK, I guess I was confused. I thought that this would be the only property required, and that if `skipOnStageFailures` was `true` then the reporter would check for failures in the stages. Is that possible?

Reply:
@alanfx master run all stages and if one failed the data won't be reported when `skipOnStageFailures = true`